src/components/input-number/input-number.tsx

```diff
@@ -756,9 +758,15 @@ export class InputNumber
       return;
     }
 
-    this.disabled
-      ? slottedActionEl.setAttribute("disabled", "")
-      : slottedActionEl.removeAttribute("disabled");
+    if (this.disabled) {
+      if (slottedActionEl.getAttribute("disabled") == null) {
```

Comment:
nitpick: Would `!slottedActionEl.hasAttribute("disabled")` read better here?

Reply:
`""` means true for attributes but it's falsy in js